src/kOS/Screen/TermWindow.cs

```diff
@@ -364,13 +385,47 @@ void TerminalGui(int windowId)
                 ShowCharacterByAscii((char)1, screen.CursorColumnShow, screen.CursorRowShow, currentTextColor);
             }
             GUI.EndGroup();
+            
+            GUI.Label(new Rect(windowRect.width/2-40,windowRect.height-20,100,10),screen.ColumnCount+"x"+screen.RowCount);
 
+            CheckResizeDrag(); // Has to occur before DragWindow or else DragWindow will consume the event and prevent drags from being seen by the resize icon.
+            GUI.DragWindow();
         }
```

Comment:
I'm becoming a bit unhappy about how long the TerminalGui() method (above) is getting.  But I was unwilling to split it up just yet, until we're certain everything is working just right the way it is.  It can be broken into smaller pieces later.